node/node_bridge_test.go

```diff
@@ -42,100 +22,14 @@ func TestBridgeLifecycle(t *testing.T) {
 	ctx, cancel := context.WithCancel(context.Background())
 	t.Cleanup(cancel)
 
-	err = node.Start(ctx)
-	require.NoError(t, err)
-
-	ctx, cancel = context.WithCancel(context.Background())
-	t.Cleanup(cancel)
-
-	err = node.Stop(ctx)
-	require.NoError(t, err)
-}
-
-// TestBridge_P2P_Streams tests the ability for Bridge nodes to communicate
-// directly with each other via libp2p streams.
-func TestBridge_P2P_Streams(t *testing.T) {
```

Comment:
No need to have this test here. Swamp does a good job of ensuring our nodes can talk to each other.